### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by miekof**
* i18n add ja structure
* Translation of partial documents

### CODE REVIEW

File: docs_source_files/content/CONTRIBUTING.ja.md [hunk 1]
```diff
@@ -8,62 +8,38 @@ disableToc: true
 日本語は話せますか？プルリクエストをして翻訳を手伝ってください!
```
Comment (diemol) [inline docs_source_files/content/CONTRIBUTING.ja.md]: @miekof could you please remove the notice? The page is already fully translated.

File: docs_source_files/content/_index.ja.md [hunk 2]
```diff
@@ -1,35 +1,23 @@
 ---
-title: "The Selenium Browser Automation Project"
+title: "Seleniumブラウザー自動化プロジェクト"
 ---
 {{% notice info %}}
```
Comment (diemol) [inline docs_source_files/content/_index.ja.md]: @miekof could you please remove the notice? The page is already fully translated.

File: docs_source_files/content/introduction/_index.ja.md [hunk 3]
```diff
@@ -9,6 +9,6 @@ weight: 2
 日本語は話せますか？プルリクエストをして翻訳を手伝ってください!
```
Comment (diemol) [inline docs_source_files/content/introduction/_index.ja.md]: @miekof could you please remove the notice? The page is already fully translated.

**Review by diemol — CHANGES_REQUESTED** [review]
Thanks for your contribution @miekof! I left a couple of comments.

**Review by diemol — APPROVED** [review]
Thank you @miekof!

File: docs_source_files/content/introduction/_index.ja.md (hunk 3 above)
Comment (miekof) [reply docs_source_files/content/introduction/_index.ja.md]: Removed notifications.

**Pull request merged**